performance.php

```diff
@@ -14,3 +14,4 @@
 require_once __DIR__ . '/performance/do-pings.php';
 require_once __DIR__ . '/performance/media.php';
 require_once __DIR__ . '/performance/plugins.php';
+require_once __DIR__ . '/performance/class-media-library-caching.php';
```

Comment:
Let's do some slight re-organization of the files

- Call the class  `Mime_Types_Caching`
- Store it at `/performance/class-mime-types-caching.php`
- But remove the require from this `performance.php` file and instead:
  - Load this new class file in the existing `/performance/media.php`
  - Move the add_action() that initiates this class inside `/performance/media.php`, perhaps with a new function declared in this file that loads the class instance and runs init(). This will make it simpler for customers to just unhook it all, and we can do away with the `vip_cache_mime_types` filter then. The class won't need to be concerned with that level of detail.

Reply:
These changes were implemented.